app/src/main/res/layout/activity_registerform.xml

```diff
@@ -11,13 +11,14 @@
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_marginStart="37dp"
-        android:layout_marginBottom="172dp"
+        android:layout_marginTop="48dp"
+        android:layout_marginEnd="16dp"
         android:onClick="registerBtn_OnClick"
         android:text="register"
-        app:layout_constraintBottom_toTopOf="@+id/guideline6"
         app:layout_constraintEnd_toStartOf="@+id/guideline4"
-        app:layout_constraintHorizontal_bias="0.241"
+        app:layout_constraintHorizontal_bias="0.344"
```

Comment:
in some lines you added new value or change those, why? Is there any specific reason for that?

Reply:
Some buttons/textView were not well placed, at least on my screen